### ISSUE TITLE
Error caused by uk/time_types.h

### ISSUE THREAD

**Issue by chenzq-98** [unikraft/lib-newlib#29]
In file included from /data1/deepl/workdir/libs/lib-newlib/include/sys/_types.h:37:0,
                 from /data1/deepl/workdir/apps/nginx/build/libnewlibc/origin/newlib-2.5.0.20170922/newlib/libc/include/machine/endian.h:5,
                 from /data1/deepl/workdir/apps/nginx/build/libnewlibc/origin/newlib-2.5.0.20170922/newlib/libc/include/sys/param.h:10,
                 from /data1/deepl/workdir/libs/lib-newlib/include/sys/param.h:4,
                 from /data1/deepl/workdir/unikraft/include/uk/essentials.h:48,
                 from /data1/deepl/workdir/unikraft/lib/ukdebug/include/uk/print.h:41,
                 from /data1/deepl/workdir/unikraft/plat/common/x86/trace.c:34:
/data1/deepl/workdir/libs/lib-newlib/include/uk/_types.h:44:10: fatal error: uk/time_types.h: No such file or directory
 #include <uk/time_types.h>
          ^~~~~~~~~~~~~~~~~

failed to compile the nginx and other libs are ok.

**Comment by istr** [comment]
See https://github.com/unikraft/unikraft/commit/2d4dea9a4f3a173d57f4738187eda29f534d26f3

This might need to be removed here as well.

**Comment by istr** [comment]
@czq693497091 how did you try to build that?

A build using `kraft configure -p kvm -m x86_64`, `kraft prepare`, `kraft build` works for me with a fresh, clean checkout of all dependencies.

**Comment by wattake** [comment]
I also got this error when using the `make` and `qemu-system-x86_64` tools instead of the `kraft` tool.

**Comment by istr** [comment]
@wattake yes. It seems that the `make` based build is not up to speed with `kraft`. If I get that right, support for `libnew` will be dropped. See https://github.com/unikraft/lib-redis/pull/7#pullrequestreview-1169451049
